program/rust/src/tests/test_upd_aggregate.rs

```diff
@@ -134,10 +158,6 @@ fn test_upd_aggregate() {
         assert_eq!(price_data.agg_.conf_, 55);
         assert_eq!(price_data.num_qt_, 2);
         assert_eq!(price_data.timestamp_, 2);
-        assert_eq!(price_data.prev_slot_, 1000);
```

Comment:
I understand why you've deleted these (because upd_aggregate no longer updates them). However, after making this change, we no longer have a test for the prev_price updates.

I think you need to add an equivalent test back somewhere else, probably in test_upd_price or test_upd_price_v2